src/ckeditor/ckeditor.component.spec.ts

```diff
@@ -4,12 +4,11 @@
  */
 
 import waitUntil from 'wait-until-promise';
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { CKEditorComponent } from './ckeditor.component';
 import {
 	fireDragEvent,
 	mockDropEvent,
-	mockPasteEvent,
```

Comment:
This helper can be removed from `test.tools.js` file as it isn't used anywhere else.